src/Documentation/doc/advanced/corporation/optimal-selling-price-market-ta2.md

```diff
@@ -0,0 +1,108 @@
+# Optimal selling price - Market-TA2
+
+## Market price and markup limit
+
+Market price:
+
+- Material: `material.marketPrice`.
+- Product: `product.productionCost`. This value is based on `ProductMarketPriceMult`, input materials' `MarketPrice` and `Coefficient`.
+  - $n = {Number\ of\ input\ materials}$
+  - $ProductMarketPriceMult = 5$
+
+$$ProductMarketPrice = ProductMarketPriceMult\ast\sum_{i = 1}^{n}{MaterialMarketPrice_i\ast MaterialCoefficient_i}$$
+
+Markup limit:
+
+- Material:
+
+$$MaterialMarkupLimit = \frac{MaterialQuality}{MaterialMarkup}$$
+
+- Product:
+
+$$ProductMarkupLimit = \frac{Max(ProductEffectiveRating,0.001)}{ProductMarkup}$$
+
+## Maximize sales volume
+
+Define:
+
+$$ExpectedSalesVolume = \frac{ProducedUnits}{10}$$
+
+- This means we want to sell all produced units.
+
+In cycle's SALE state, game calculates `MaxSalesVolume` of material/product. If we set price too high, `MaxSalesVolume` is penalized. In order to maximize profit, we have to set the highest possible price while `MaxSalesVolume` is still equals to `ExpectedSalesVolume`. This is what Market-TA2 does.
+
+Calculation of material and product is pretty similar, so I'll call them "item" and use 1 formula.
+
+`MaxSalesVolume` is the product of 7 multipliers:
+
+- Item multiplier:
+  - Material: $ItemMultiplier = MaterialQuality + 0.001$
+  - Product: $ItemMultiplier = 0.5\ast(ProductEffectiveRating)^{0.65}$
+- Business factor:
+
+$$BusinessProduction = 1 + office.employeeProductionByJob["Business"]$$
+
+$${BusinessFactor = (BusinessProduction)}^{0.26} + \left( \frac{BusinessProduction}{1000} \right)$$
+
+- Advert factor:
+  - $AwarenessFactor = (Awareness + 1)^{IndustryAdvertisingFactor}$
+  - $PopularityFactor = (Popularity + 1)^{IndustryAdvertisingFactor}$
+
+$$RatioFactor = \begin{cases}Max(0.01,\frac{Popularity + 0.001}{Awareness}), & Awareness \neq 0 \newline 0.01, & Awareness = 0 \end{cases}$$
+
+$$AdvertFactor = (AwarenessFactor\ast PopularityFactor\ast RatioFactor)^{0.85}$$
+
+- Market factor:
+
+$$MarketFactor = Max\left( 0.1,\frac{Demand\ast(100 - Competition)}{100} \right)$$
```

Comment:
Here and possibly elsewhere: You can save some vertical space/make some quantities less shrunk by eliminating needless `\frac`s. Specifically, division by `100` can be replaced by `* 0.01`, etc.